### PR TITLE
Change name and description

### PR DESCRIPTION
**Pull request by LuisaE**
Use "official" name: Wiki Addition Viewer: Add insights to your Wikipedia experience.

### CODE REVIEW

File: docs/literature.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
                         <div class="col-lg-8 col-xxl-6">
                             <div class="text-center my-5">
                                 <h1 class="fw-bolder mb-3">Literature review</h1>
-                                <p class="lead fw-normal text-muted mb-4">We are basing our browser extension on several research papers.</p>
+                                <p class="lead fw-normal text-muted mb-4">We are basing our browser extension on two research papers.</p>
```
Comment (RichartE) [inline docs/literature.html:38]: Should we link the papers?

File: wikichange/package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "wikichange",
-    "description": "Annontate changes on Wikipedia pages",
-    "version": "0.0.1",
+    "name": "Wiki Addition Viewer",
+    "description": "Add insights to your Wikipedia experience",
+    "version": "1.0.0",
     "homepage": "http://chrome.google.com/webstore",
-    "author": "ENTER YOUR NAME HERE",
+    "author": "Comps team",
```
Comment (RichartE) [inline wikichange/package.json]: Should we add Carleton?

File: wikichange/package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
     "repository": {
         "type": "git",
         "url": "ENTER GIT REPO URL"
```
Comment (RichartE) [inline wikichange/package.json]: Should we link our repository?

File: wikichange/src/manifest.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "__MSG_appName__",
+    "name": "Wiki Addition Viewer",
```
Comment (RichartE) [inline wikichange/src/manifest.json]: Why do we need to change these ones?

**Review by RichartE — COMMENTED** [review]
Looks good just a couple of questions

File: docs/literature.html (hunk 1 above)
Comment (LuisaE) [reply docs/literature.html:38]: Good idea!

File: wikichange/src/manifest.json (hunk 4 above)
Comment (LuisaE) [reply wikichange/src/manifest.json]: We don't, reverted the change

**Review by sukritsangvong — COMMENTED** [review]
looks good too me if @RichartE approves

**Comment by LuisaE** [comment]
@etiennerichart can I merge?

**Review by sukritsangvong — APPROVED** [review]

**Pull request merged**